Source/WebKit/WebProcess/WebPage/WebFrame.cpp

```diff
@@ -178,6 +179,7 @@ Ref<WebFrame> WebFrame::createRemoteSubframe(WebPage& page, WebFrame& parent, We
 WebFrame::WebFrame(WebPage& page, WebCore::FrameIdentifier frameID)
     : m_page(page)
     , m_frameID(frameID)
+    , m_inspectorTarget(makeUniqueRef<WebFrameInspectorTarget>(*this))
```

Comment:
is there any downside to doing this initialization lazily and using a RefPtr? I'm concerned about the overhead for the vast majority of users who aren't inspecting.